test/test_chunking.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import pytest
+
+from RAiDER.delayFcns import chunk,makeChunksFromInds,makeChunkStartInds
+
+# The purpose of these tests is to verify the chunking done on the query points during is
+# done correctly for various dimensions
+
+def checkChunks(chunk1, chunk2):
+    check = []
+    for k1,L in enumerate(chunk2):
+        for k2,arr in enumerate(L):
+            if np.allclose(arr, chunk1[k1][k2]):
+                check.append(True)
+            else:
+                check.append(False)
+    return all(check)
+
+def test_makeChunkStartInds1D():
+    chunkStartInds = [(0,), (1,), (2,), (3,), (4,)]
+    assert chunkStartInds == makeChunkStartInds((1,),(5,))
+
+def test_makeChunkStartInds2D():
+    chunkStartInds = [(0, 0), (0, 1), (3, 0), (3, 1), (6, 0), (6, 1)]
+    assert chunkStartInds == makeChunkStartInds((3,1), (9,2))
+
+def test_makeChunkStartInds3D():
+    chunkStartInds = (
+        [(0, 0, 0),
+	 (0, 2, 0),
+	 (0, 4, 0),
+	 (0, 6, 0),
+	 (2, 0, 0),
+	 (2, 2, 0),
+	 (2, 4, 0),
+	 (2, 6, 0)] 
+	)
+    assert chunkStartInds == makeChunkStartInds((2,2,16), (4,8,16))
+
+def test_makeChunksFromInds1D():
+    in_shape = (4,)
+    chunkSize = (2,)
+    ind = [(0,), (2,)]
+    chunks = [[np.array([0, 1])], [np.array([2, 3])]]
+    assert checkChunks(chunks, makeChunksFromInds(ind,chunkSize,in_shape)) 
+
+def test_makeChunksFromInds2D():
+    in_shape = (4,4)
+    chunkSize = (2,2)
+    ind = [(0, 0), (0, 2), (2, 0), (2, 2)]
```

Comment:
No need for all these variables, they just clutter up the code. If you want to make the function call more readable, use keyword arguments.